engine/orchestration/src/main/java/org/apache/cloudstack/engine/orchestration/NetworkOrchestrator.java

```diff
@@ -2435,7 +2438,7 @@ protected void removeNic(final VirtualMachineProfile vm, final NicVO nic) {
             for (final NetworkElement element : networkElements) {
                 if (providersToImplement.contains(element.getProvider())) {
                     if (s_logger.isDebugEnabled()) {
-                        s_logger.debug("Asking " + element.getName() + " to release " + nic);
+                        s_logger.debug(String.format("Asking %s to release %s, according to %s", element.getName(), nic, nic.getReservationStrategy()));
```

Comment:
according to (????) %s

Reply:
the reservation strategy?

Reply:
yeah

Reply:
and this ?

Reply:
not sure what you mean to ask here @weizhouapache ?

Reply:
@DaanHoogland  like this ?

```suggestion
                        s_logger.debug(String.format("Asking %s to release %s, according to the reservation strategy %s", element.getName(), nic, nic.getReservationStrategy()));
```